documentation/src/components/layout.ts

```diff
@@ -10,23 +10,67 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { html, CSSResultArray, property } from 'lit-element';
+import {
+    html,
+    CSSResultArray,
+    property,
+    LitElement,
+    PropertyValues,
+} from 'lit-element';
 import './side-nav';
 import layoutStyles from './layout.css';
-import { RouteComponent } from './route-component';
+import '@spectrum-web-components/theme';
+import { Color, Scale } from '@spectrum-web-components/theme';
+import { Dropdown } from '@spectrum-web-components/dropdown';
+import '@spectrum-web-components/dropdown';
+import '@spectrum-web-components/menu';
+import '@spectrum-web-components/menu-item';
 
-export class LayoutElement extends RouteComponent {
+const SWC_THEME_COLOR_KEY = 'swc-docs:theme:color';
+const SWC_THEME_SCALE_KEY = 'swc-docs:theme:scale';
+const COLOR_LIGHT = 'light';
+const SCALE_MEDIUM = 'medium';
+const DEFAULT_COLOR = (window.localStorage
+    ? localStorage.getItem(SWC_THEME_COLOR_KEY) || COLOR_LIGHT
+    : COLOR_LIGHT) as Color;
+const DEFAULT_SCALE = (window.localStorage
+    ? localStorage.getItem(SWC_THEME_SCALE_KEY) || SCALE_MEDIUM
+    : SCALE_MEDIUM) as Scale;
+
+export class LayoutElement extends LitElement {
     public static get styles(): CSSResultArray {
         return [layoutStyles];
     }
 
+    @property({ attribute: false })
+    public color: Color = DEFAULT_COLOR;
+
     @property({ type: Boolean })
     public open = false;
 
+    @property({ attribute: false })
+    public scale: Scale = DEFAULT_SCALE;
+
     toggleNav() {
         this.open = !this.open;
     }
 
+    private updateColor(event: Event) {
+        this.color = (event.target as Dropdown).value as Color;
+    }
+
+    private updateScale(event: Event) {
+        this.scale = (event.target as Dropdown).value as Scale;
+    }
+
+    private onClickLabel(event: { target: HTMLElement }) {
+        const { target } = event;
```

Comment:
Some hacked manual click forwarding for the `label` representing the `sp-dropdown` it's not _awesome_, but we're gonna have to have a larger conversation in https://github.com/adobe/spectrum-web-components/issues/475 as to how to support labels, so this allows us to move forward in delivering this UI in the interim.

Reply:
Do we need a *TODO* comment here about this so that we can find it again, or reference this code in the related issue?

Reply:
Good call, I'll add one to the related CSS that was added for the same reason.